.travis.yml

```diff
@@ -1,4 +1,5 @@
 # Travis CI Configuration File
```

Comment:
do we still need Travis if we have a php linting action?

Reply:
Unit tests! 😁